src/components/swiper/swiper.spec.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+import Swiper from './swiper.vue';
+import { mount } from '@vue/test-utils';
+
+describe('Component swiper', () => {
+    it('is an object', () => {
+        expect(Swiper).to.be.an('object');
+        expect(Swiper).to.be.not.empty;
+    });
+
+    it('passes attrs props to swiper config', () => {
```

Comment:
the unit test in this case is ok since swiper is already well tested. Otherwise you should write much more tests :)

Reply:
that's what I've thought. First I wanted to write some more tests but realized that it doesn't make any sense :)